shuttle/templates/schedules/driver_logs.html

```diff
@@ -1,72 +1,123 @@
+                                            {% if shuttle_logs[j]['location'] != None %}
+                                               {{ shuttle_logs[j]['location'] }}
+                                            {% endif %}
+                                        </td>
+                                        <td class="check-in">
+                                            {% if shuttle_logs[j]['arrival_time'] != None %}
+                                                {{ shuttle_logs[j]['arrival_time'] }}
+                                            {% endif %}
+                                        </td>
+                                        <td class="check-out">
+                                            {% if shuttle_logs[j]['departure_time'] != None %}
+                                                {{ shuttle_logs[j]['departure_time'] }}
+                                            {% endif %}
+                                        </td>
+                                    {% endif %}
+                                    </tr>
+                                {% endfor %}
+                            </tbody>
+                        </table>
+                    </div>
+                </div>
+            </div>
+        </div>
+    {% endfor %}
+
+
+
+
     <hr>
+
+
     <h4>Driver Break Logs</h4>
-    <div class="card">
-        <table class="table table-hover">
-            <thead>
-            <tr>
-                <th scope="col">Driver</th>
-                <th scope="col">Break Time In</th>
-                <th scope="col">Break Time Out</th>
-            </tr>
-            </thead>
-            <tbody>
-                <tr>
-                    <td class="driver">*Insert Driver here*</td>
-                    <td class="check-in-break">*Insert time in here*</td>
-                    <td class="check-out-break">*Insert time out here*</td>
-                </tr>
-                <tr>
-                    <td scope="driver">*Insert Driver here*</td>
-                    <td class="check-in-break">*Insert time out here*</td>
-                    <td class="check-out-break">*Insert time out here*</td>
-                </tr>
-                <tr>
-                    <td scope="driver">*Insert Driver here*</td>
-                    <td class="check-in-break">*Insert time in here*</td>
-                    <td class="check-out-break">*Insert time out here*</td>
-                </tr>
-            </tbody>
-        </table>
-    </div>
 
+    {% for i in range(date_list|length) %}
+        <div id="accordion">
+            <div class="card">
+                <div class="card-header" id="heading-tw">
+                    <h5 class="mb-0">
+                        <button class="btn btn-link" type="button" data-toggle="collapse" data-target="#break-log-collapse-{{ i }}" aria-expanded="true" aria-controls="break-log-collapse-{{ i }}">
+                            {{ date_list[i] }}
```

Comment:
Can you make this a selector instead of multiple buttons you can see.

So a <select> with <option>s that are the dates that whenever you select a date it loads in all the shuttle logs and break logs for that date.

Also could you format those dates that you click on/select to only have the date and not the time. Currently they have a time of 00:00:00 and that isn't helpful.

Reply:
You could probably pull the log/driver log html out to a jinja macro and just load that in with the selected date.

https://github.com/betheluniversity/writing-center-python/blob/master/writing_center/templates/macros/login-to-google-calendar.html

Here is an example of a macro in our writing center repo.

Reply:
Also if dates don't have any logs don't include the dates on the selector.